src/main/java/com/gw/tools/ProcessTool.java

```diff
@@ -737,7 +737,7 @@ public String all_history(String pid, boolean ignore_skipped) {
 	
 	public String all_history(String pid) {
 
-		return all_history(pid, true);
+		return all_history(pid, false);
```

Comment:
As of now we are not explicity showing skipped status in the history. I beleive it is set that way for a reason. This change will make it show always. I think only when the filter is active we should be showing skipped.

Reply:
@ZihengSun might have a better understanding for hiding the skipped processes by default
skipped processes might make it look cluttered is my take on this and we should not be showing it by default.

Reply:
I have made the required changes. Please check.

Reply:
Changes can be viewed in [Extended the functionality of the filtering processes](https://github.com/ESIPFed/Geoweaver/pull/475/commits/5ba619a62de6331f50e84d78935c46679138c9a8)